### PR TITLE
rkolmos/react: Configure webpack/react make a dummy component

### PR DESCRIPTION
**Pull request by ghost**
Added a webpack dev server
Added react.
Created a dummy component and wired up the /index page to render it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "build:dev": "webpack --mode=development",
+    "build:prod": "webpack --mode=production",
+    "watch": "webpack --watch",
+    "serve": "webpack serve"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/JarheadStever/mp-climb-stat.git"
+  },
+  "author": "jstever",
```
Comment (JarheadStever) [inline package.json]: jarheadstever author

File: package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "my-webpack-project",
+  "version": "1.0.0",
+  "description": "My webpack project",
+  "main": "app.js",
+  "scripts": {
+    "build": "webpack --mode=production",
+    "build:dev": "webpack --mode=development",
+    "build:prod": "webpack --mode=production",
```
Comment (JarheadStever) [inline package.json]: unnecessary

File: package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    "watch": "webpack --watch",
```
Comment (JarheadStever) [inline package.json:9]: unnecessary

File: src-js/app.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+                       name="input"
+                       onChange={(event) => this.setState( { input: event.target.value })}
+                />
+                <input type="submit" value="Submit"/>
+            </form>
+        </div>;
+    }
+}
+
+ReactDOM.render(<Component/>, document.getElementById('content'));
```
Comment (JarheadStever) [inline src-js/app.js]: newline

**Review by JarheadStever — CHANGES_REQUESTED** [review]

**Comment by JarheadStever** [comment]
you have failed the empire

File: package.json (hunk 3 above)
Comment (ghost) [reply package.json:9]: If it ever fails to watch it is probably good to be able to specify this explicitly

**Review by JarheadStever — COMMENTED** [review]
LGTM

**Review by JarheadStever — APPROVED** [review]

**Pull request merged**